ChangeLog.d/issue3819.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fix incorrect seed generation when a nonce is included
+     in mbedtls_ctr_drbg_reseed_internal.
```

Comment:
This is not just an incorrect calculation: it's a security reduction. Effectively the nonce was all-bits-zero, so calling `mbedtls_ctr_drbg_set_nonce_len` did not increase the security strength of the initial seeding (and in particular the security strength was not as defined by NIST). Please make the description more precise and change the classification to “Security”.